5-Heroku/server/controllers/todoController.js

```diff
@@ -0,0 +1,49 @@
+const {
+    modelGetTodoById,
+    modelGetAllTodos,
+    modelPostTodo,
+    modelDeleteTodo,
+    modelUpdateTodo} = require('../models/todoModel');
+const {v4: uuidv4} =  require('uuid');
+
+const getTodoById = (req, res) => {
+    modelGetTodoById(req, (todo) => {
+        res.status(200).json(todo)
+    });
+}
+const postTodo = (req, res) => {
+    modelPostTodo(req, () => {
+        res.sendStatus(201);
+    })
+}
+
+const getAllTodos = (req, res) => {
+    const cookieId = req.cookies['user_id'];
+    if (!cookieId) {
+        const cookieId = uuidv4();
+        res.cookie('user_id', cookieId, {httpOnly: true, secure: false });
```

Comment:
the req object is well an object :)
so once u read the userId you can write something like
req.userId = userId
and then everywhere in the code that u want to get it from it will be nicer then req.cookies.userId